src/core/library.c

```diff
@@ -1322,6 +1337,26 @@ QuicLibraryGetGlobalParam(
         Status = QUIC_STATUS_SUCCESS;
         break;
 
+#ifdef QUIC_USE_RAW_DATAPATH
```

Comment:
Should this also be wrapped in `#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES`?

Reply:
No, I don't see any precedents. The code in this file for `QUIC_PARAM_GLOBAL_VERSION_SETTINGS` isn't wrapped in `#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES`.